src/app/intelligence/intelligence.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import {Observable} from "rxjs";
+import {Store} from "@ngrx/store";
+import * as fromRoot from '../reducers';
+import {IntelligenceService} from "../intelligence.service";
+
+@Component({
+  selector: 'app-intelligence',
+  templateUrl: './intelligence.component.html',
+  styleUrls: ['./intelligence.component.css']
+})
+export class IntelligenceComponent implements OnInit {
+  wholequery$: Observable<any>;
+  actions: Array<any> = [];
+  answer: any;
+  constructor(private store: Store<fromRoot.State>, private intelligence: IntelligenceService) {
+    this.wholequery$ = store.select(fromRoot.getwholequery);
+    this.wholequery$.subscribe(data => {
+      this.intelligence.getintelligentresponse(data.query).subscribe(res => {
+        console.log(res, res.answers , res.answers[0]);
+        if (res && res.answers && res.answers[0].actions) {
+          this.actions = res.answers[0].actions;
+          for (let action of this.actions){
+            if (action.type === 'answer' && action.mood !== 'sabta') {
+               this.answer = action.expression;
```

Comment:
this is some random mood name given in susi when the results are not sure, hence whenever mood is sabta, didn't display the result.